src/main/java/io/stargate/sgv2/jsonapi/service/operation/filters/table/codecs/JSONCodec.java

```diff
@@ -178,6 +179,24 @@ static <JavaT> ToCQL<JavaT, JavaT> unsafeIdentity() {
       return (toCQLType, value) -> value;
     }
 
+    /**
+     * Method for constructing exception-handling String-to-CQL-type converter, given a function
+     * that converts a String to the target CQL type.
+     */
+    static <JavaT, CqlT> ToCQL<JavaT, CqlT> safeFromString(Function<JavaT, CqlT> function) {
+      return (toCQLType, value) -> {
+        try {
+          return function.apply(value);
+        } catch (IllegalArgumentException | DateTimeException e) {
```

Comment:
Selecting exceptions known to be thrown for content problem, but trying to avoid catching general `Exception` to let things like NPEs flow through to be caught at higher levels (as they are not problems with input value but something different).